src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -29,23 +29,17 @@ export interface GraphWrapperProps extends State {
 	onMoreCommits?: (limit?: number) => void;
 }
 
-// Copied from original pushed code of Miggy E.
-// TODO: review that code as I'm not sure if it is the correct way to do that in Gitlens side.
-// I suppose we need to use the GitLens themes here instead.
 const getCssVariables = (mixedColumnColors: CssVariables | undefined): CssVariables => {
-    const body = document.body;
-    const computedStyle = window.getComputedStyle(body);
-
-	return {
-        '--app__bg0': computedStyle.getPropertyValue('--color-background'),
-        '--panel__bg0': computedStyle.getPropertyValue('--graph-panel-bg'),
-		'--text-selected': computedStyle.getPropertyValue('--color-foreground'),
-		'--text-normal': computedStyle.getPropertyValue('--color-foreground--85'),
-		'--text-secondary': computedStyle.getPropertyValue('--color-foreground--65'),
-		'--text-disabled': computedStyle.getPropertyValue('--color-foreground--50'),
-		'--text-accent': computedStyle.getPropertyValue('--color-link-foreground'),
-		'--text-inverse': computedStyle.getPropertyValue('--vscode-input-background'),
-		'--text-bright': computedStyle.getPropertyValue('--vscode-input-background'),
+    return {
+        '--app__bg0': 'var(--color-background)',
```

Comment:
The graph component expects resolved values in the `cssVariables` input property

Reply:
@ericf-axosoft does it ever use the color-string not in CSS?

Reply:
@miggy-e we have already talked about that (see https://github.com/gitkraken/vscode-gitlens/pull/2105#discussion_r926844100). The graph expects already computed variables.

Maybe we won't have any problem with this specific variable but we should but we have to keep consistency. We cannot compute some yes and others no.  

Also remember that we should validate colors on the component side (see https://gitkraken.atlassian.net/browse/GITLENS-283 and https://github.com/gitkraken/vscode-gitlens/pull/2105#discussion_r929151546). How can we validate the colors if they are not computed?